core-utils/src/main/java/com/anytypeio/anytype/core_utils/ext/AndroidExtension.kt

```diff
@@ -182,7 +182,8 @@ fun Int.addDot(): String = "$this."
 
 fun EditText.multilineIme(action: Int) {
     setRawInputType(
-        InputType.TYPE_TEXT_FLAG_MULTI_LINE
+        InputType.TYPE_CLASS_TEXT
```

Comment:
i've just quickly tested this solution. the drawback, on my phone, is that pressing enter no longer creates a new text block but instead inserts newline character. could you confirm the same behavior on your side?

Reply:
Yes, I can. I had noticed that as well and that’s why I drafted the PR. I believe that it should replicate the same behavior as the desktop version (I use Windows), which is `Return -> New block` and `Shift+Return -> New line in the same block`. I just gotta take a look in what is the default Android solution for that. I’ll see what I can do

Reply:
Okay, got it fixed. And soft breaks still work if using an external keyboard. The thing is that the change supposedly removes multiline support (besides soft break). I could re-enable it where it is needed but I haven't found any place that had actual multiline support in the v0.24.4 release

Reply:
does your fix allow to create a new text block when pressing ENTER? will check this soon, thank you.

Reply:
Here's a demo

https://github.com/anyproto/anytype-kotlin/assets/69634269/8ef90f7f-d4e7-453c-a797-ad427ab788ec

Reply:
looks like working solution. we'll test it and get back to you, thanks

Reply:
> in the v0.24.4 release

actually, multiline support is needed when, for instance, instead of creating a new block, you (force)insert a linebreak, so that you have one block with several line breaks. on Android, we should display these line breaks. could you test on your side also?

Reply:
Besides `Shift+Return` with a keyboard (which I tested and work), how can I force a linebreak? Pasting and copying?

Reply:
> > in the v0.24.4 release
> 
> actually, multiline support is needed when, for instance, instead of creating a new block, you (force)insert a linebreak, so that you have one block with several line breaks. on Android, we should display these line breaks. could you test on your side also?

i've just checked, it works!

Reply:
I have auto-correction working, and all the other functionality. We'll test it for a couple of days and consider the solution, but at first glance, it's a very useful fix. Thank you @BrayanDSO